config/jobs/kubernetes-sigs/cluster-api-operator/cluster-api-operator-presubmits-main.yaml

```diff
@@ -92,10 +125,18 @@ presubmits:
         args:
         - runner.sh
         - ./scripts/ci-test.sh
+        resources:
+          requests:
+            cpu: "1"
+            memory: "2Gi"
+          limits:
+            cpu: "1"
+            memory: "2Gi"
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-operator
       testgrid-tab-name: capi-operator-pr-test-main
   - name: pull-cluster-api-operator-e2e-main
+    cluster: eks-prow-build-cluster
```

Comment:
@ameukam After the merge, this job started failing in https://github.com/kubernetes-sigs/cluster-api-operator/pull/143. Looking at the build logs in https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-operator/143/pull-cluster-api-operator-e2e-main/1668941993025212416 I don't find anything useful and just:

```
Job execution failed: Pod pending timeout.
Failed to open /logs/process-log.txt: open /logs/process-log.txt: no such file or directory
```

Reply:
Looks like an issue with the build cluster. Let me ping folx on Slack.

Reply:
Thanks, please tag me in the thread if possible

Reply:
+1 I'd love to follow along too. There have been a few merged jobs transitioning that have had issues so far.